BangazonDB.sql

```diff
@@ -0,0 +1,552 @@
+(PurchaseDate, Model, DecommissionDate, Condition)
+select '2/13/15', 'Dell', null, 'fair';
+
+Insert into Computer
+(PurchaseDate, Model, DecommissionDate, Condition)
+select '2/13/15', 'Dell', null, 'fair';
+
+Insert into Computer
+(PurchaseDate, Model, DecommissionDate, Condition)
+select '2/13/15', 'Dell', null, 'fair';
+
+Insert into Computer
+(PurchaseDate, Model, DecommissionDate, Condition)
+select '12/12/12', 'Dell', '6/21/17', 'decommissioned';
+
+Insert into Computer
+(PurchaseDate, Model, DecommissionDate, Condition)
+select '1/22/13', 'HP', '5/18/17', 'decommissioned';
+					
+CREATE TABLE EmployeeComputer (
+Id	integer NOT NULL PRIMARY KEY IDENTITY,
+EmployeeId	integer NOT NULL,
+ComputerId	integer NOT NULL,
+DateAssigned		varchar(80) not null,
+DateTurnedIn		varchar(80),
+Constraint FK_Employee_Computer FOREIGN KEY(EmployeeId) REFERENCES Employee(Id),
+Constraint FK_Computer FOREIGN KEY(ComputerId) REFERENCES Computer(Id)
+);
+					
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 1, 1, c.PurchaseDate, null
+from Computer c where c.Id = 1;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 2, 2, c.PurchaseDate, null
+from Computer c where c.Id = 2;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 3, 3, c.PurchaseDate, null
+from Computer c where c.Id = 3;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 4, 4, c.PurchaseDate, null
+from Computer c where c.Id = 4;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 5, 5, c.PurchaseDate, null
+from Computer c where c.Id = 5;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 6, 6, c.PurchaseDate, null
+from Computer c where c.Id = 6;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 7, 7, c.PurchaseDate, null
+from Computer c where c.Id = 7;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 8, 8, c.PurchaseDate, null
+from Computer c where c.Id = 8;
+
+Insert into EmployeeComputer
+(EmployeeId, ComputerId, DateAssigned, DateTurnedIn)
+select 9, 9, c.PurchaseDate, null
+from Computer c where c.Id = 9;
+
+					
+Create table ProductType (
+Id	integer not null primary key IDENTITY,
+Name	varchar(80) not null
+);
+
+Insert into ProductType
+(Name)
+select 'Toys';
+
+Insert into ProductType 
+(Name)
+select 'Home & Garden';
+
+Insert into ProductType
+(Name) 
+select 'Health & Beauty';
+
+Insert into ProductType
+(Name) 
+select 'Food';
+
+CREATE TABLE Product (
+Id	integer NOT NULL PRIMARY KEY IDENTITY,
+ProductTypeId	integer NOT NULL,
+Price	integer NOT NULL,
+Title		varchar(80) not null,
+ProdDesc		varchar(80) not null,
+Quantity		integer not null,
+Constraint FK_ProductType FOREIGN KEY(ProductTypeId) REFERENCES ProductType(Id)
```

Comment:
A product needs a foreign key of the customer who is selling it.

Reply:
It's giving me an error that the Customer table is invalid now.